.dir-locals.el

```diff
@@ -0,0 +1,16 @@
+;; Emacs settings for cardano-wallet
```

Comment:
Suggestion: I think I'd prefer if this file was renamed to `.dir-locals.el.example` or something similar. It's super useful to mine for ideas, but quickly diverges between programmers, and I can't imagine dealing with the merge conflicts being fun, you'd have to stash your local changes every time you pull then pop them afterwards?

That way we could contribute ideas to the example file without having to worry about breaking other's configs.

Reply:
Yes you're right.
I will remove it, and use its text as an example in our documentation.
It would be nice to have editors working properly "out of the box."
With VSCode, you can add stuff like recommended plugins and settings into the `.vscode` directory.
But perhaps this is too difficult to do for emacs.